test/integration/util.go

```diff
@@ -55,6 +55,7 @@ func flag(f string) string {
 }
 
 func getProjectNameForTest() string {
+	projectNameRandom = uuid.New().String()
```

Comment:
how did it work before?

Reply:
This is effect the deletion of tests in the end. revert this change